src/main.rs

```diff
@@ -909,6 +953,27 @@ async fn run() -> Result<(), Error> {
     )
     .await?;
 
+    info!("Starting quota watcher");
+    let ec3_client = ec2_client.clone();
```

Comment:
ec3?

Reply:
Explained in a comment in https://github.com/MaterializeInc/k8s-eip-operator/pull/145/commits/0517b786ab24688d7ef563fd7744e36acc94b885 -- need to do this to move the client into the interval task.

Reply:
cool, thanks